build/buildspec.docker.dev.yml

```diff
@@ -23,14 +23,14 @@ phases:
       - DATE=$(date "+%F,%T")
       - VERSION=$(echo $GIT_BRANCH | tr "/" .)
       # Build Open source version
-      - CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -installsuffix cgo -ldflags="-s -w -X main.version=$VERSION -X main.commit=$CODEBUILD_RESOLVED_SOURCE_VERSION -X main.buildDate=$DATE -X main.builtBy=codebuild-$CODEBUILD_BUILD_ID "
-      - docker build -t rudderlabs/develop-rudder-server:$VERSION -f build/Dockerfile-aws .
+      - CGO_ENABLED=1 GOOS=linux go build -race -mod vendor -a -installsuffix cgo -ldflags="-w -X main.version=$VERSION -X main.commit=$CODEBUILD_RESOLVED_SOURCE_VERSION -X main.buildDate=$DATE -X main.builtBy=codebuild-$CODEBUILD_BUILD_ID "
```

Comment:
We have moved to using makefile now for building. Please rebase this branch and reimplement this using Makefile.